.github/scripts/spawn_agent.py

```
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
```

Ruff (0.13.1)

Shebang is present but file is not executable

(EXE001)